benchmark/babel-rewrite.js

```diff
@@ -0,0 +1,47 @@
+const babelParser = require('@babel/parser')
```

Comment:
I'm not sure if we must add the Licensing header in benchmark files.

Reply:
Added